File: problemsets/Codeforces/Python/A1136.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
n=int(input())
a=[int(input().split()[1]) for _ in [0]*n ]
k=int(input())
print(n-sum(x<k for x in a))

File: problemsets/Codeforces/C++/A1136.cpp
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
#include <bits/stdc++.h>

using namespace std;

int main() {
  int n, k, l, r[100];
  cin >> n;
  for (int i = 0; i < n; i++)
    cin >> l >> r[i];
  cin >> k;
  cout << n-(lower_bound(r, r+n, k)-r);
}
